src/mongo/services/schemaService.ts

```diff
@@ -103,7 +103,7 @@ export default class SchemaService {
 		})
 	}
 
-	private setSchemaForDocument(parent: string, document: any, schema: JSONSchema): void {
+	private setSchemaForDocument(parent: string, document: {}, schema: JSONSchema): void {
```

Comment:
Based on the line right below this, it feels like `document` could be multiple different types, including array. I think `any` is more accurate than `{}`

Reply:
This is another case where I might prefer to just disable "no-any" for the file - especially since we inherited this code and we're pretty unfamiliar with it

Reply:
Object is what the SDK function that it called was declared as expecting. However, I'm good to just disable it.